docs/website/01_framework/observer.md

```diff
@@ -5,11 +5,12 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 
 # Oberserver and Observable
 
-To solve situations like "I want to write an output when the loadstep is completed", the observer pattern is implemented in Ikarus.
-To understand it and use it in your implementation, you need to understand three things: ``IObservable``, ``IObserver`` and Messages.
+To write output messages when a load step is completed, the observer pattern is implemented in Ikarus.
```

Comment:
"something is finished or similar" the observer notion is not restricted to load steps